src/statmodels.jl

```diff
@@ -423,6 +423,29 @@ Return all parameters of a model.
 params(model) = error("params is not defined for $(typeof(model))")
 function params! end
 
+"""
+    vif(model, param)
```

Comment:
```suggestion
    vif(model::StatisticalModel, param)
```

Reply:
Would it make sense to restrict this to `::RegressionModel`? AFAIK the VIF is only used for linear regression, right?

Reply:
I think it's used for any model falling within the generalized linear model and there might even be meaningful definitions for a broader class of models. But it's much easier to start narrow and broaden, so I'm fine restricting this to `RegressionModel` for now.